nest-cli.json

```diff
@@ -1,7 +1,7 @@
 {
   "$schema": "https://json.schemastore.org/nest-cli",
   "collection": "@nestjs/schematics",
-  "sourceRoot": "src",
+  "sourceRoot": "/",
```

Comment:
out of curiosity - what was happening that you had to make this change?

Reply:
It was an attempt to fix the `npm run build` problem we have talked about. Forgot to undo this